Dockerfile.dapper

```diff
@@ -62,19 +59,55 @@ RUN mkdir /usr/tmp && \
     mv /usr/tmp/helm /usr/bin/helm_v3 && \
     chmod +x /usr/bin/kustomize
 
-RUN curl -sLf ${!K3S_DOWNLOAD_URL} -o /usr/bin/k3s && \
-    chmod +x /usr/bin/k3s && \
-    mkdir -p /go/src/github.com/rancher/rancher/.kube && \
+# Set up K3s: copy the necessary binaries from the K3s image.
+COPY --from=rancher/k3s:v1.22.4-k3s1 \
+    /bin/blkid \
+    /bin/charon \
+    /bin/cni \
+    /bin/conntrack \
+    /bin/containerd \
+    /bin/containerd-shim-runc-v2 \
+    /bin/ethtool \
+    /bin/ip \
+    /bin/ipset \
+    /bin/losetup \
+    /bin/pigz \
+    /bin/runc \
+    /bin/swanctl \
+    /bin/which \
+    /bin/aux/xtables-legacy-multi \
```

Comment:
Do we actually need the `iptables`/`xtables` binaries for K3s to run within the Rancher container? 

I have a feeling the explicit use of the legacy binaries may actually cause problems if on a system where the `ip_tables` module is not enabled.

If we actually need the `iptables` binaries, I would recommend just copying the entire `aux` folder out of `/bin/aux` into the `rancher` image so as to preserve the ability to perform iptables module detection (`ip_tables` vs `nftables`) which we do today.

Reply:
Copying symlinks out of Docker images is kinda broken; it copies the file instead of remaining a symlink. We ended up doing this to keep this as a symlink.

Reply:
I have been a little concerned about which iptables-related packages to copy over. Currently, in `/bin/aux/`, I see lots of `ip6tables*`, `arptables*`, `ebtables*` symlinks. What if some are needed? In my testing, I have not seen any errors with just having a few select `iptables` and `ip6tables` binaries, but maybe in some scenarios K3s might actually need one that's not been copied over?